pynest/nest/tests/test_get_set.py

```diff
@@ -311,51 +312,59 @@ def test_get_JSON(self):
         multi_sd = nest.Create('spike_detector', 10)
 
         # Single node, literal parameter
-        self.assertEqual(single_sd.get('start', output='json'), 0.0)
+        self.assertEqual(json.loads(
```

Comment:
I think it makes more sense that instead of dereferencing the JSON output, you check that you get correct JSON format. Because `get(..., output='json')` should return JSON, so for me it is then strange that we convert the JSON to a Python object and test against the corresponding object. Makes more sense in my mind to test the JSON that we get against excepted JSON. This goes for all the tests.

Reply:
I tried that solution first, but as the order of elements in dictionaries is not conserved, the JSON output string has a chance to become different from the reference JSON string.

Reply:
Ah, then I understand.